includes/class.llms.view.manager.php

```diff
@@ -360,6 +346,39 @@ public function scripts() {
 
 	}
 
+
+	/**
+	 * Determine whether or not the view manager should be added to the WP Admin Bar
+	 *
+	 * The view manager is only displayed when the following criteria is met:
+	 * + The current user must have a role that is allowed to bypass LifterLMS restrictions
+	 * + Must be viewing a single course, lesson, membership, or quiz or the LifterLMS checkout page.
+	 *
+	 * @since [version]
+	 *
+	 * @return boolean
+	 */
+	protected function should_display() {
+
+		$display = false;
+
+		if ( llms_can_user_bypass_restrictions( get_current_user_id() ) ) {
+			global $post;
+			$display = is_admin() || is_post_type_archive() || ! $post || ( ! is_llms_checkout() && ! in_array( $post->post_type, array( 'course', 'lesson', 'llms_membership', 'llms_quiz' ), true ) ) ? false : true;
```

Comment:
What do you think is more readable? (not joking, I'm going to take your answer as cs for the future) 

```suggestion
			$display = ! ( is_admin() || is_post_type_archive() || ! $post || ( ! is_llms_checkout() && ! in_array( $post->post_type, array( 'course', 'lesson', 'llms_membership', 'llms_quiz' ), true ) ) );
```

Reply:
I think both of them are probably a little bit unreadable... but the leading `!` is making me have to think too hard about this.

Reply:
thanks.